WalletWasabi.Fluent/Controls/TileControl.axaml

```diff
@@ -1,37 +1,44 @@
+        xmlns:c="clr-namespace:WalletWasabi.Fluent.Controls">
   <Design.PreviewWith>
-    <controls:TileControl TileSize="Large">
-      <Panel Width="330" Height="150" Background="Blue" />
-      <controls:TileControl.LargeSizeContent>
-        <Panel Width="330" Height="330" Background="Blue" />
-      </controls:TileControl.LargeSizeContent>
-      <controls:TileControl.WideSizeContent>
-        <Panel Width="660" Height="330" Background="Blue" />
-      </controls:TileControl.WideSizeContent>
-    </controls:TileControl>
+    <c:TileControl Height="140" Width="310" Title="Tile">
+      <c:TileControl.BottomContent>
+        <Button VerticalAlignment="Stretch">Bye</Button>
+      </c:TileControl.BottomContent>
+      <Button VerticalAlignment="Bottom">hello</Button>
+    </c:TileControl>
   </Design.PreviewWith>
 
-  <Style Selector="controls|TileControl">
-    <!-- Set Defaults -->
+  <Style Selector="HeaderedContentControl.tile">
     <Setter Property="Template">
       <ControlTemplate>
-        <Border Name="PART_MainBorder"
-                CornerRadius="2" BorderThickness="1"
-                Margin="6"
-                Background="{DynamicResource TileRegionColor}"
-                BorderBrush="{DynamicResource TileBorderColor}">
-          <Panel>
-            <Border Padding="16">
-              <ContentPresenter Name="PART_ContentPresenter"
-                                Background="{TemplateBinding Background}"
-                                Padding="{TemplateBinding Padding}"
-                                VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
-                                HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}" />
-            </Border>
-          </Panel>
-        </Border>
+        <DockPanel>
+          <ContentPresenter DockPanel.Dock="Top" Content="{TemplateBinding Header}"
+                            Classes="h8 bold"
+                            ContentTemplate="{TemplateBinding HeaderTemplate}" />
+          <ContentPresenter Content="{TemplateBinding Content}" ContentTemplate="{TemplateBinding ContentTemplate}" />
+        </DockPanel>
+      </ControlTemplate>
+    </Setter>
+  </Style>
+
+  <Style Selector="c|TileControl">
+    <Setter Property="SeparatorMargin" Value="0 6"></Setter>
+    <Setter Property="BottomPartHeight" Value="22"></Setter>
+    <Setter Property="Template">
+      <ControlTemplate>
+        <HeaderedContentControl Classes="tile" Header="{TemplateBinding Title}">
+          <DockPanel>
+            <StackPanel DockPanel.Dock="Bottom" Name="PART_BottomContent" IsVisible="{TemplateBinding IsBottomContentVisible}">
+              <Separator DockPanel.Dock="Bottom" Margin="{TemplateBinding SeparatorMargin}" />
+              <Viewbox Height="{TemplateBinding BottomPartHeight}">
+                <ContentPresenter Content="{TemplateBinding BottomContent}" />
+              </Viewbox>
```

Comment:
Why is this `ViewBox`? Feels like a hack

Reply:
Was a way to ensure that bottom part is always restricted to the area it should occupy, otherwise, borders in different tiles might not align. 

I've removed it and nothing broke, so it's OK to remove it :)